<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>niftkSingleViewerControls</class>
 <widget class="QWidget" name="niftkSingleViewerControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>136</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="m_ControlPanelGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="2" column="2" rowspan="3">
    <widget class="QWidget" name="m_WindowBindingWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="m_BindWindowsLabel">
        <property name="text">
         <string>bind windows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_BindWindowCursorsCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Binds the cursor positions relative to the window size, across the windows of the same viewer.&lt;/p&gt;&lt;p&gt;If the magnification is not bound then if you pan the image or select a position then the image will be shifted in the other windows, so that the cursor planes are aligned in the adjacent windows.&lt;/p&gt;&lt;p&gt;Note that this option controls only the displayed position of the cursor, not the selected position in space. The selected position in space is &lt;span style=&quot; font-style:italic;&quot;&gt;always&lt;/span&gt; linked across the windows of the same image viewer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>cursor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_BindWindowMagnificationsCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Binds the magnification across the windows of an image viewer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>magn.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="m_WindowLayoutWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="2" column="0">
       <widget class="QRadioButton" name="m_MultiWindowRadioButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switches the window layout to show multiple windows.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="m_SagittalWindowRadioButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switches the window layout to one sagittal window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>sag</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="m_3DWindowRadioButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switches the window layout to one 3D window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>3D</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="m_CoronalWindowRadioButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switches the window layout to one coronal window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>cor</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="m_AxialWindowRadioButton">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switches the window layout to one axial window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>ax</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="m_MultiWindowComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Selects a window layout that has multiple windows.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="m_ShowOptionsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="m_WindowShowOptionsLabel">
        <property name="text">
         <string>show:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowCursorCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Shows the cursor (aka. crosshair)</string>
        </property>
        <property name="text">
         <string>cursor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowDirectionAnnotationsCheckBox">
        <property name="toolTip">
         <string>Shows the orientation directions (S, I, A, P, L, R)</string>
        </property>
        <property name="text">
         <string>directions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowPositionAnnotationCheckBox">
        <property name="toolTip">
         <string>Shows the position at the cursor</string>
        </property>
        <property name="text">
         <string>position</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowIntensityAnnotationCheckBox">
        <property name="toolTip">
         <string>Shows the intensity of the selected voxel in the visible images</string>
        </property>
        <property name="text">
         <string>intensity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_ShowPropertyAnnotationCheckBox">
        <property name="toolTip">
         <string>Shows the selected properties of the visible images</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="niftk::ClickableLabel" name="m_PropertiesForAnnotationLabel">
        <property name="text">
         <string>properties</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="m_ShowOptionsSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="5">
    <widget class="Line" name="m_WindowLayoutSeparator">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3">
    <widget class="QWidget" name="m_SlidersWidget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="m_SliceLabel">
        <property name="text">
         <string>slice:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkSliderWidget" name="m_SliceSlider" native="true">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows the index of the selected slice in the currently selected 2D window.&lt;/p&gt;&lt;p&gt;Changing the slice index changes the &lt;span style=&quot; font-style:italic;&quot;&gt;selected position&lt;/span&gt; along the dimension that is orthogonal to the plane of the currently selected orientation. The selected position is always bound across the windows of the same viewer. You can bind the selected position across the viewers by the '&lt;span style=&quot; font-style:italic;&quot;&gt;position&lt;/span&gt;' check box in the '&lt;span style=&quot; font-style:italic;&quot;&gt;bind viewers&lt;/span&gt;' section on this panel.&lt;/p&gt;&lt;p&gt;The selected position can be displayed by the &lt;span style=&quot; font-style:italic;&quot;&gt;cursor&lt;/span&gt; (aka. crosshair). Changing the slice index also changes the &lt;span style=&quot; font-style:italic;&quot;&gt;cursor position&lt;/span&gt; in the other two 2D windows of the same viewer.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="m_TimeStepLabel">
        <property name="text">
         <string>time:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkSliderWidget" name="m_TimeStepSlider" native="true">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changes the time step of the currently selected&lt;br/&gt;image viewer and also in other image viewers if&lt;br/&gt;the cursor is bound between them.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="m_MagnificationLabel">
        <property name="text">
         <string>magn.:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="ctkSliderWidget" name="m_MagnificationSlider" native="true">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Changes the magnification in the currently&lt;br/&gt;selected window, and also in other windows&lt;br/&gt;if it is bound between windows or image viewers.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="m_ShowOptionsSeparator">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="m_WindowBindingOptionsSeparator">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header location="global">ctkSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>niftk::ClickableLabel</class>
   <extends>QLabel</extends>
   <header>niftkClickableLabel_p.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
